python/ray/autoscaler/updater.py

```diff
@@ -35,13 +35,15 @@ def with_interactive(cmd):
 
 
 class KubernetesCommandRunner:
-    def __init__(self, log_prefix, namespace, node_id, auth_config,
-                 process_runner):
+    def __init__(self, log_prefix, node_id, provider, auth_config,
```

Comment:
Any reason why we're passing in the full node provider here just to access the namespace? Can we just pass in the namespace instead?

Reply:
Sure, but why do we need to change the signature for this class constructor?

Reply:
That's a good point, the constructor signature of the actual cmd runner doesn't matter, only the get_command_runner() interface.

Reply:
ok I fixed it.